openmetadata-ui/src/main/resources/ui/src/components/AddService/Steps/ConfigureService.tsx

```diff
@@ -11,96 +11,79 @@
  *  limitations under the License.
  */
 
-import { Button } from 'antd';
+import { Button, Form, Space } from 'antd';
+import { ENTITY_NAME_REGEX } from 'constants/regex.constants';
 import { t } from 'i18next';
-import React, { useRef } from 'react';
-import { errorMsg, requiredField } from '../../../utils/CommonUtils';
-import RichTextEditor from '../../common/rich-text-editor/RichTextEditor';
-import { EditorContentRef } from '../../common/rich-text-editor/RichTextEditor.interface';
-import { Field } from '../../Field/Field';
+import React from 'react';
+import { FieldProp, FieldTypes, generateFormFields } from 'utils/formUtils';
 import { ConfigureServiceProps } from './Steps.interface';
 
 const ConfigureService = ({
   serviceName,
-  description,
-  showError,
-  handleValidation,
   onBack,
   onNext,
 }: ConfigureServiceProps) => {
-  const markdownRef = useRef<EditorContentRef>();
+  const [form] = Form.useForm();
 
-  const validationErrorMsg = (): string => {
-    if (showError.name) {
-      return t('message.field-text-is-required', {
-        fieldText: t('label.service-name'),
-      });
-    }
-    if (showError.duplicateName) {
-      return t('message.entity-already-exists', {
-        entity: t('label.service-name'),
-      });
-    }
-    if (showError.delimit) {
-      return t('message.service-with-delimiters-not-allowed');
-    }
-    if (showError.nameWithSpace) {
-      return t('message.service-with-space-not-allowed');
-    }
-    if (showError.nameLength) {
-      return t('message.service-name-length');
-    }
-    if (showError.specialChar) {
-      return t('message.special-character-not-allowed');
-    }
+  const formFields: FieldProp[] = [
+    {
+      name: 'name',
+      id: 'root/name',
+      required: true,
+      label: t('label.service-name'),
+      type: FieldTypes.TEXT,
+      rules: [
+        {
+          pattern: ENTITY_NAME_REGEX,
+          message: t('message.entity-name-validation'),
+        },
+      ],
+      props: {
+        'data-testid': 'service-name',
+      },
+      placeholder: t('label.service-name'),
+      formItemProps: {
+        initialValue: serviceName,
+      },
+    },
+    {
+      name: 'description',
+      required: false,
+      label: t('label.description'),
+      id: 'root/description',
+      type: FieldTypes.DESCRIPTION,
+      props: {
+        'data-testid': 'description',
+        initialValue: '',
+      },
+      formItemProps: {
+        trigger: 'onTextChange',
+        valuePropName: 'initialValue',
+      },
```

Comment:
This should be handled along with `FieldTypes.DESCRIPTION`  I feel

Reply:
Thanks, that make sense, I will take care of this in a separate patch as it will impact other places as well.